pkg/imgpkg/bundle/bundle.go

```diff
@@ -45,11 +46,23 @@ type ImagesMetadata interface {
 	FirstImageExists(digests []string) (string, error)
 }
 
+// GraphNode Node information of a Bundle
+type GraphNode struct {
```

Comment:
GraphNode does not really indicate what it is. is it some type of content within a bundle? what would be a better name for this?

Reply:
I had called it PullGraphNode before this commit but it felt too specific and we might use this structure in the future to keep some resemblance to the structure of the bundle. By structure, I am saying the bundles associated with the current bundle.

Reply:
Since it was in the `bundle` package I assumed that `bundle.GraphNode` would have enough information. What do you think?